_maps/map_files/templates/dungeons/north_bunker_2.dmm

```diff
@@ -1085,8 +1087,9 @@
 /area/f13/bunker)
 "xS" = (
 /obj/structure/rack,
-/obj/effect/spawner/lootdrop/f13/weapon/gun/tier10,
 /obj/effect/spawner/lootdrop/f13/armor/tier5,
+/obj/effect/spawner/lootdrop/f13/weapon/gun/random_high,
+/obj/effect/spawner/lootdrop/f13/weapon/gun/random_high,
```

Comment:
I see an increase of spawners on this dungeon.
From four fixed spawns (giving loot of tier 5, 8, 8 and 10) to nie spawns, five of which giving loot of tiers 1 to 7, plus four giving stuff of tiers 5 to 10 or unique stuff.

Are you sure this is what you were trying to accomplish?

Pebbles did some new loot tables to replace the existing ones. Are these those?

Reply:
I replaced the static tier spawners on most dungeons with the random version of their respective spawner. With some of the mid-tier spawners, instead of placing down a random high tier I placed down two random normal tiers to give a bigger chance of a better gun but still generally being lower tier.

basically, I added more RnG spawners in place of the static ones and added a few more in an attempt to still make clearing the dungeon somewhat worth it.

Reply:
The end result is a buff, though, not a nerf.